### ISSUE TITLE
Things like `fmt.println()` escape sandbox

### ISSUE THREAD

**Issue by AlbinoGeek** [d5/tengo#342]
Since there's no captured `stdout`, `stdin` or related -- `fmt.println()` and friends can be seen "escaping the sandbox" so to say.

Maybe I missed something?

## Golang Source

```golang
var (
	req      ExecuteRequest
	script   *tengo.Script
	compiled *tengo.Compiled
	err      error
	ctx      context.Context
	cancel   func()
	output   interface{}
)

script = tengo.NewScript([]byte(req.Body))

if compiled, err = script.Compile(); err != nil {
	return
}

ctx, cancel = context.WithTimeout(context.TODO(), time.Second*5)
defer cancel()

if err = compiled.RunContext(ctx); err != nil {
	return
}

vars = compiled.GetAll()
	
// How do I capture the output !?
```

#273 is very related.

---

Unfortunately I am running these in parallel, so I can't easily hijack the owning stdout/stdin for scripts.

**Comment by AlbinoGeek** [comment]
If no one else is working on this particular issue, I could fork and update stdlib.fmt to use buffers/writers instead of real system handles.

**Comment by geseq** [comment]
Please feel free to work on it.

**Comment by AlbinoGeek** [comment]
:) Got this working on my end fairly easily, currently working on sandboxing other libraries; I wonder however:

Would it be preferable these were alternatives to stdlib instead? (e.g.: alternate import)

The performance of writing to a pipe in memory v.s. actual stdout varies wildly depending on your use-case, for sure -- but other libraries are not so cut-and-dry as to the implication(s) of sandboxing them (think: `os`.)

Which reminds me... opening a separate ticket.

**Comment by geseq** [comment]
Might be a good idea to make them alternatives then. @d5 thoughts?

**Comment by d5** [comment]
Yes. Please create a new module. It's better for backward compatibility, and it gives more options to people with different requirements.
